package.json

```diff
@@ -13,6 +13,7 @@
     "start": "ng serve",
     "build": "ng build",
     "test": "ng test ngx-onboarding",
+      "test-coverage-report": "ng test ngx-onboarding --code-coverage --watch=false",
```

Comment:
Ass discussed locally: We can use Webstorm features to generate coverage local and the build server already uses the coverage flag.